server/lib/routes/get-verify-email.js

```diff
@@ -0,0 +1,47 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const got = require('got');
+
+const logger = require('mozlog')('server.get-verify-email');
+const config = require('../configuration');
+
+module.exports = function (uid, code, options) {
+
+  return {
+    method: 'get',
+    path: '/verify_email/',
+    process: function (req, res, next) {
+      req.url = '/';
+      console.log('items', req.query);
+
+      let data = {
+        code: req.query.code,
+        uid: req.query.uid
+      };
+
+      let options = req.query.options;
+      if (options) {
+        if (options.service) {
+          data.service = options.service;
+        }
+
+        if (options.reminder) {
+          data.reminder = options.reminder;
+        }
+      }
+
+      got.post(config.get('fxaccount_url') + '/v1/recovery_email/verify_code', {
```

Comment:
See if we can use a template literal here: https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Template_literals instead of `+`